README.md

```diff
@@ -1,2 +1,110 @@
-# terraform-google-log-export
-A Cloud Foundation Toolkit Module: Opinionated Google Cloud Platform project creation and configuration with Shared VPC, IAM, APIs, etc.
+# Terraform Logsink Module
+
+## Requirements
+### Terraform plugins
+- [Terraform](https://www.terraform.io/downloads.html) 0.10.x
+- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v1.8.0
+
+### Configure a Service Account
+In order to execute this module you must have a Service Account with the following:
+
+#### Roles
+The service account with the following roles:
+- roles/logging.configWriter on the logsink project (to create logsinks)
+- roles/serviceAccount.admin on the logsink project (to create a service account for the log sinks)
+- roles/iam.admin on the logsink project (to grant permissions to the service account)
+- roles/serviceusage.admin on both the logsink and the destination project (to enable API for destinations)
+- roles/storage.editor on the destination project (if logsink destination is a Cloud Storage bucket)
+- roles/pubsub.editor on the destination project (if logsink destination is a Cloud Pub/Sub topic)
+- roles/bigquery.dataEditor on the destination project (if logsink destination is a BigQuery table)
+
+
+### Enable API's
+In order to operate with the Service Account you must activate the following API's on the base project where the Service Account was created:
+
+- Cloud Resource Manager API - cloudresourcemanager.googleapis.com

```

Comment:
It would be helpful to have a script to do this, but not needed at first. Opened #2 to address.